proto/babylon/btcstkconsumer/v1/params.proto

```diff
@@ -10,4 +10,8 @@ option go_package = "github.com/babylonlabs-io/babylon/x/btcstkconsumer/types";
 message Params {
   option (amino.name) = "babylon/x/btcstkconsumer/v1/Params";
   option (gogoproto.equal) = true;
-}
\ No newline at end of file
+
+  // permissioned_integration is a flag to enable permissioned integration, i.e.,
+  // requiring governance proposal to approve new integrations.
+  bool permissioned_integration = 1;
```

Comment:
Can we set a default here? Just curious.

Reply:
It's set [here](https://github.com/babylonlabs-io/babylon/blob/de84d8817da424189fa5110ac00c2e3dc1333949/x/btcstkconsumer/types/params.go#L17-L18)